client/app/dbaas/dbaas-metrics/header/metrics-header.controller.js

```diff
@@ -1,13 +1,13 @@
 (() => {
     class MetricsHeaderCtrl {
-        constructor ($state, $stateParams, $translate, METRICS_ENDPOINTS, ovhDocUrl) {
+        constructor ($state, $stateParams, $translate, METRICS_ENDPOINTS, ovhDocUrl, URLS) {
```

Comment:
Here is the preferred order for parameters in constructors:
* First, the services starting with a $
* Then our own services, without *
* Finally, the constants

So here is how it should be:
```js
constructor ($state, $stateParams, $translate, ovhDocUrl, METRICS_ENDPOINTS, URLS) {
	this.$state = $state;
    this.$stateParams = $stateParams;
    this.$translate = $translate;

    this.ovhDocUrl = ovhDocUrl;

    this.METRICS_ENDPOINTS = METRICS_ENDPOINTS;
    this.URLS = URLS;

	this.serviceName = $stateParams.serviceName;
    this.guides = {};
}
```